src/hooks/useUnistyles.ts

```diff
@@ -66,6 +72,14 @@ export const useUnistyles = () => {
         return subscription.remove
     }, [])
 
+    if (unistylesContext !== undefined) {
```

Comment:
that's interesting I need to test it out

Reply:
I think we also need to add the context as a dependency in the effect? So that nothing breaks if the context is rendered conditionally 
Which is a very rare scenario but could happen

Reply:
yeah, I think its required, please add it